### PR TITLE
add a download for feedback and upload previus feedback

### PR DESCRIPTION
**Pull request by aabughosh**
[CNFCERT-490]Auto-populate comments in HTML parser based on JSON
a download feedback button that downloading the feedback that we have now as a json file and by pressing upload feedback we can upload a json file of feedback

### CODE REVIEW

File: script/results.html [hunk 1]
```diff
@@ -62,6 +62,11 @@
   <button class="w3-button w3-right w3-light-grey" id="download" type="submit" onclick=download()
     hidden="hidden">Download Results Feedback</button>
 
+    <button class="w3-button w3-right w3-light-grey" id="downloadjson" type="submit" onclick=downloadjson()
+    hidden="hidden">Download Feedback as a JSON file</button>
+    <p hidden="hidden" id ="UploadFeedback">
+      Upload Feedback File: <input type="file" id="feedbackFile" size="40">
```
Comment (edcdavid) [inline script/results.html:68]: The upload seems to work only the first time for me, after this there is no effect

File: script/results.html [hunk 2]
```diff
@@ -596,6 +606,18 @@
...
+    function fillFeedback(input){
+
+      for ( key in input) { 
+        // copy previus daata
+        document.getElementById(key).textContent = document.getElementById(key).value;
+    
+        document.getElementById(key).textContent = input[key]
+        document.getElementById(key).value=input[key]
+      }
+
```
Comment (rdavid) [inline script/results.html:618]: No need for the new line.

**Review by edcdavid — APPROVED** [review]

File: script/results.html [hunk 3]
```diff
@@ -62,6 +62,11 @@
   <button class="w3-button w3-right w3-light-grey" id="download" type="submit" onclick=download()
     hidden="hidden">Download Results Feedback</button>
 
+    <button class="w3-button w3-right w3-light-grey" id="downloadjson" type="submit" onclick=downloadjson()
```
Comment (edcdavid) [inline script/results.html:65]: result.html updated, regenerate the result-embed.html (same script used to regenerate catalog)

Comment (aabughosh) [reply script/results.html:65]: i did rebase

Comment (aabughosh) [reply script/results.html:65]: i did rebase

**Review by sebrandon1 — APPROVED** [review]

**Comment by ramperher** [comment]
@aabughosh can you add some instructions about how can we use this new feature? I mean, what do we need to do to upload previous feedback?

**Comment by aabughosh** [comment]
> @aabughosh can you add some instructions about how can we use this new feature? I mean, what do we need to do to upload previous feedback?

yah need to upload previous feedback and u can download it also

**Pull request merged**